env/staging/ecs/task-definitions/metrics.json

```diff
@@ -21,6 +21,10 @@
         }
       },
       "environment": [
+        {
+          "name": "MASKED_DATA",
```

Comment:
I believe this should be `MASK_DATA` for the env var name:
https://github.com/cds-snc/covid-alert-metrics/blob/main/app.py#L7

Reply:
Fixed.